steamtinkerlaunch

```diff
@@ -4096,6 +4098,8 @@ function saveCfg {
 			echo "SDLUSEWAYLAND=\"$SDLUSEWAYLAND\""
 			echo "## $DESC_STLRAD_PFTST"
 			echo "STLRAD_PFTST=\"$STLRAD_PFTST\""
+			echo "## $DESC_STEAMDECK_AUTOUP"
```

Comment:
This puts the option in the Per-Game Config File and not the Global Config. It should be moved to `saveCfgglobal`.

[This PR](https://github.com/sonic2kk/steamtinkerlaunch/pull/935/files#diff-77c6d4bb1bf7c1988ff5a068c856b6c24e113663f2e61cb3a2c98723c0fafc04R3637-R3640) might serve as a good reference for adding a global config value. I should really add comments for this too now that I've thought about it :-)

Reply:
Nice catch ! Fixed it in my last commit :)